private/bufpkg/bufcheck/internal/bufcheckserver/internal/bufcheckserverhandle/breaking.go

```diff
@@ -1827,7 +1827,7 @@ func handleBreakingMessageSameRequiredFields(
 			// we attach the error to the added required field
 			responseWriter.AddProtosourceAnnotation(
 				requiredField.Location(),
-				nil, // TODO:figure out the correct against location for this
```

Comment:
I should have removed this before the merge to bufplugin, but there is no AgainstLocation in this particular case.